backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/UserPermission.java

```diff
@@ -2,12 +2,16 @@
 
 /**
  * Permissions that a user may hold, usually via their {@link OrganizationRole}.
+ *
+ * DO: Keep updated with UserPermission in schema.graphqls
  */
 public enum UserPermission {
     READ_PATIENT_LIST,
+    READ_PATIENT_INCL_DEL_LIST,
```

Comment:
This is the only permission that isn't a plain-English term—can we find one that would work? I agree that `READ_PATIENT_LIST_INCLUDING_DELETED_ONES` is not the right answer, but maybe `READ_DELETED_PATIENTS` or `READ_DELETED_PATIENT_LIST`?